nodejs/src/table.rs

```diff
@@ -328,16 +328,36 @@ impl Table {
             .map(IndexConfig::from)
             .collect::<Vec<_>>())
     }
+
+    #[napi]
+    pub async fn index_stats(
+        &self,
+        index_uuid: String,
+    ) -> napi::Result<Either<IndexStatistics, ()>> {
+        // We use Either<T, ()> instead of `Option` to avoid returning null. We'd rather return 'undefined'
```

Comment:
why is this?

Reply:
So this may be personal preference, and I'm fine with using `null` here. But in general, I think it's much easier for consumers to only have to deal with one `None` type instead of having to handle both `undefined` and `null`.